time stamp
int32 type
float64 x
float64 y
float64 z
